koku/api/report/test/tests_queries.py

```diff
@@ -141,3 +147,124 @@ def test_group_data_by_list_missing_units(self):
                     ],
                 }
                 self.assertEqual(expected, out_data)
+
+
+class AWSQueryHandlerTest(IamTestCase):
```

Comment:
Moved to wrong file, should be here:  koku/koku/api/report/test/aws/tests_queries.py

https://github.com/project-koku/koku/pull/1552#discussion_r357439804

The above discussion shows up strangely in my browser now the source file was removed so I'm just adding a comment here for clarity.

Reply:
moved to koku/api/report/test/aws/tests_queries.py